Tests/JWTKitTests/X5CTests.swift

```diff
@@ -119,6 +119,59 @@ final class X5CTests: XCTestCase {
+        """
+
+        struct StoreKitPayload: ValidationTimePayload {
+            func verify(using _: JWTAlgorithm) async throws {}
+
+            let data: DataClass
+            let notificationUUID: String
+            let signedDate: Date
+            let notificationType: String
+        }
+
+        struct DataClass: Codable {
+            let appAppleId: Int
+            let environment, bundleId: String
+        }
+
+        let verifier = try X5CVerifier(rootCertificates: [cert])
+
+        let jsonDecoder = JSONDecoder()
+        jsonDecoder.dateDecodingStrategy = .millisecondsSince1970
+
+        var payload: StoreKitPayload?
+        do {
+            payload = try await verifier.verifyJWS(data, as: StoreKitPayload.self, jsonDecoder: jsonDecoder)
+        } catch {
+            XCTFail("Failed with error: \(error.localizedDescription)")
+        }
+
+        if let payload = payload {
```

Comment:
You can use `XCTUnwrap` for this - also this is 5.9 so it should just be `if let payload`!